cli/azd/main.go

```diff
@@ -58,29 +55,6 @@ func main() {
 
 	ts := telemetry.GetTelemetrySystem()
 
-	assignmentEndpoint := "https://default.exp-tas.com/exptas49/b80dfe81-554e-48ec-a7bc-1dd773cd6a54-azdexpws/api/v1/tas"
```

Comment:
I move it away from main to create the desired effect of only incurring the ~200ms endpoint performance penalty if the args passed does correspond to an actual azd command.